MVP/Resources/Translations.no.resx

```diff
@@ -0,0 +1,456 @@
+    </data>
+    <data name="setting_apptheme_title" xml:space="preserve">
+        <value>App-tema</value>
+    </data>
+    <data name="setting_apptheme_description" xml:space="preserve">
+        <value>Denne appen skal bli brukt i både lys å mørk modus. Som standard velger den hva enheten er satt til. Hvis et av temaene foretrekkes, et bestemmt tema kan velges her.</value>
+    </data>
+    <data name="logout" xml:space="preserve">
+        <value>Logg ut</value>
+    </data>
+    <data name="tabs_activities" xml:space="preserve">
+        <value>Aktiviteter</value>
+    </data>
+    <data name="tabs_statistics" xml:space="preserve">
+        <value>Statistikk</value>
+    </data>
+    <data name="tabs_badges" xml:space="preserve">
+        <value>Merker</value>
+    </data>
+    <data name="tabs_settings" xml:space="preserve">
+        <value>Innstillinger</value>
+    </data>
+    <data name="contributionform_title_add" xml:space="preserve">
+        <value>Legg til bidrag</value>
+    </data>
+    <data name="contributionform_title_edit" xml:space="preserve">
+        <value>Rediger bidrag</value>
+    </data>
+    <data name="languagepicker_title" xml:space="preserve">
+        <value>Språk</value>
+    </data>
+    <data name="contributiondetails_title" xml:space="preserve">
+        <value>Aktivitetsdetaljer</value>
+    </data>
+    <data name="field_activity_type" xml:space="preserve">
+        <value>Aktivitetstype</value>
+    </data>
+    <data name="field_url" xml:space="preserve">
+        <value>URL</value>
+    </data>
+    <data name="field_title" xml:space="preserve">
+        <value>Tittel på aktivitet</value>
+    </data>
+    <data name="field_primary_contribution_area" xml:space="preserve">
+        <value>Primærbidragsområde</value>
+    </data>
+    <data name="field_additional_contribution_areas" xml:space="preserve">
+        <value>Ytterlige bidragsområder</value>
+    </data>
+    <data name="field_activity_date" xml:space="preserve">
+        <value>Aktivitetsdato</value>
+    </data>
+    <data name="field_description" xml:space="preserve">
+        <value>Beskrivelse</value>
+    </data>
+    <data name="field_visibility" xml:space="preserve">
+        <value>Synlighet</value>
+    </data>
+    <data name="button_delete_contribution" xml:space="preserve">
+        <value>Slett bidrag</value>
+    </data>
+    <data name="about_title" xml:space="preserve">
+        <value>Om denne appen</value>
+    </data>
+    <data name="about_why_title" xml:space="preserve">
+        <value>Hvorfor ble denne appen bygget?</value>
+    </data>
+    <data name="about_why_description" xml:space="preserve">
+        <value>Det har alltid vært et behov for Microsoft MVPs å styre samfunnsaktivitene sine på en enklere måte. Nettportalen fungerer fint, men å ha noen lignende i lomma for å lage aktiviter i farta er helt rått. Som er hva dette prosjektet er for!</value>
```

Comment:
Ratt translates to raw everywhere I look, probably slang but I'd love to know :D

Reply:
Yeah I'll update it to 'cooked' when the app is finished :D